Library/Formula/connect.rb

```diff
@@ -0,0 +1,21 @@
+class Connect < Formula
+  desc "provides SOCKS and HTTPS proxy support to SSH."
+  homepage "https://bitbucket.org/gotoh/connect/wiki/Home"
+  url "https://bitbucket.org/gotoh/connect/get/1.104.tar.gz"
+  sha256 "6ca5291ea8e4a402d875a5507ac78d635373584fd1912f2838b93e03b99730c8"
+
+  def install
+    # make throws an error looking for the version of windows shell, can be ignored
+    # gcc will produce warnings but the application will work as expected
```

Comment:
We could just `inreplace` out the `WINVER := $(shell ver)` line to stop the check happening.